source/particle/world.cc

```diff
@@ -163,26 +161,39 @@ namespace aspect
     {
       signals.post_set_initial_state.connect(std_cxx11::bind(&World<dim>::setup_initial_state,
                                                              std_cxx11::ref(*this)));
+
       signals.pre_refinement_store_user_data.connect(std_cxx11::bind(&World<dim>::register_store_callback_function,
                                                                      std_cxx11::ref(*this),
+                                                                     false,
                                                                      std_cxx11::_1));
+      signals.pre_serialization_store_user_data.connect(std_cxx11::bind(&World<dim>::register_store_callback_function,
```

Comment:
This is not quite correct, but I agree it is confusing. These functions are at the same time callback functions (to the SimulatorSignals), and `register...` functions, because their main purpose is to register the actual callback functions (called `store_tracers` and `load_tracers`) into the Triangulation by calling `register_data_attach` and `notify_ready_to_unpack`. I feel that `store_callback_function` is more misleading than their current name, but I am happy to rename them if we find a more descriptive name.